<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDateTime</class>
 <widget class="QDialog" name="DialogDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/wheel-0.png</normaloff>:/res/wheel-0.png</iconset>
  </property>
  <widget class="QLabel" name="lblDateTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date/Time</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDateUp">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDateDown">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ПОТВЪРДИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ОТКАЗ</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>80</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>65</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTimeUp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTimeDown">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
